lib/customs.js

```diff
@@ -52,6 +52,9 @@ module.exports = function (log, error) {
           }
           throw error.requestBlocked()
         }
+        if (result.suspect) {
+          request.app.isSuspiciousRequest = true
+        }
```

Comment:
FWIW I'm not thrilled about tunneling state between two helper libs by stashing items into `request.app`, but I couldn't come up with an approach that was obviously better.